docs/source/user-guide/tutorials/build-nltk.rst

```diff
@@ -0,0 +1,76 @@
+=================================
+Building a conda package for NLTK
+=================================
+
+The Natural Language Toolkit (NLTK) is a platform used for building Python 
+programs that apply statistical natural language processing (NLP) to human 
+language data. It can be difficult to install, but is easy when you build this 
+NLTK conda package.
+
+This package also includes all of the NLTK data sets, approximately 8 GB of
+data.
+
+Download the following files: https://gist.github.com/danielfrg/d17ffffe0dc8ed56712a0470169ff546
+(click the Download  ZIP button on the top right).
+
+Unzip the downloaded file and rename the downloaded directory to
+``/nltk-with-data/``.
+
+If not already installed, install conda build::
+
+    conda install conda-build
+
+Change directory to one directory above the nltk-with-data directory with
+``cd``.
+
+Run conda build for different Python versions that you need, selecting the
+packages for the platform and OS you are running the command on:
+
+.. code-block:: python
+
+    conda build nltk-with-data --python 2.7
```

Comment:
We could do a matrix here to make it faster.  I can show you how to do that - it's a bit more advanced, but not hard.